helm/values.yaml

```diff
@@ -52,9 +52,14 @@ serviceSelectorLabels:
 
 configMap:
   name: attribute-service-config
+  dataStoreType: "mongo"
```

Comment:
Can we keep the name of dataStoreType consistent with `mongodb` and `postgres` below.
Either change, the dataStoreType to `mongodb` or change the below config key to `mongo`